### ISSUE TITLE
Provide Homebrew tap

### ISSUE THREAD

**Issue by AlekSi** [FerretDB/FerretDB#2469]
### What should be done?

After #2468, we should provide Homebrew tap.

Docs: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

### Where?

https://github.com/FerretDB/homebrew-ferretdb

### Definition of Done

`brew tap ferretdb/ferretdb && brew install ferretdb` works.

**Comment by AlekSi** [comment]
Duplicate of #1951
